docs/user-guide/troubleshooting.rst

```diff
@@ -293,3 +293,18 @@ defined by the `working_directory` config in `server` section of `/etc/pulp/serv
 default value is `/var/cache/pulp`. Any user defined path needs to be owned by user and group
 `apache`. If running with SELinux in Enforcing mode, the path also needs to have
 `system_u:object_r:pulp_var_cache_t` security context. 
+
+Celery terminates the worker in case of sync cancellation.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the syncronization of the repo is cancelled, the worker process exits immediately with sys.exit().
+Then celery spawns right away another worker, so further tasks are normally picked up and executed.
```

Comment:
s/Then celery spawns right away another worker,/A new worker process is created immediately,/